service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/returnedcase/ReturnedCaseTemplate.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.model.notify.returnedcase;
+
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
+
+@Getter
+@Setter
+public class ReturnedCaseTemplate extends SharedNotifyTemplate {
```

Comment:
Does this require the fields provided here?

Reply:
Yes. caseUrl, localAuthority and firstRespondentName.

Reply:
That is a lot of other things that aren't required in `ReturnedCaseTemplate`, thoughts on extracting those into a different class to be used